src/Cake.Sonar.Test/Cake.Sonar.Test.csproj

```
@@ -1,19 +1,19 @@
 ﻿<Project Sdk="Microsoft.NET.Sdk">
 
   <PropertyGroup>
-    <TargetFramework>net5.0</TargetFramework>
+    <TargetFramework>net6.0</TargetFramework>
```

```suggestion
    <TargetFrameworks>net6.0;net5.0;netcoreapp3.1</TargetFrameworks>
```

I'd suggest running tests on all frameworks that are being targeted by the Cake.Sonar addin

---

Done.
I also upgraded to Cake 2.0 and fixed the NuGet package creation.